src/engraving/style/style.cpp

```diff
@@ -43,7 +43,7 @@ using namespace mu::engraving;
 
 const PropertyValue& MStyle::value(Sid idx) const
 {
-    if (idx == Sid::NOSTYLE) {
+    if (idx == Sid::NOSTYLE || size_t(idx) >= m_values.size()) {
```

Comment:
In that additional case IMHO some logging should happen, to maybe catch the real root cause

Reply:
Mmmm, probably we should just address the root cause now instead of committing trash like this patch.

Reply:
- If only the root cause were known

Reply:
This doesn't seem like an overly mysterious bug to solve. Hopefully the musescore team will repro at some point and it'll appear clear to whoever maintains this.

Reply:
The mystery is where whan and why that apperent stack corruption happens. That's why I think loggong (and asserts in debuge builds) mivht be usefull

Reply:
_If_ we merge this, it should be an assertion, because this situation is so wrong. Normally, this check wouldn't make sense because a valid Sid is never greater/equal than `m_values.size()`. 
Our IF_ASSERT_FAILED macro might be useful here:
```c++
    IF_ASSERT_FAILED(size_t(idx) < m_values.size()) { return; }
```

However, adding this assertion won't bring us any further, because hitting an assertion failure here or a read-outside-bounds a few lines later doesn't make much of a difference. So I really hope we'll ever be able to reproduce it to find out what's really going on.